frontend/src/components/contactsList.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import { CreateAContactForm } from './contactsForm'
+import axios from 'axios'
+
+interface Contact {
+    id: string
+    firstName: string
+    lastName?: string
+    email?: string
+    phoneNumber?: string
+    notes?: string
+    source?: string
+    ownerID: string
+    relationships?: UserContactRelationship[]
+    importantDateEvent?: ImportantDateEvent[]
+  }
+
+interface UserContactRelationship {
+    id: string
+    relationshipType: string
+  }
+
+interface ImportantDateEvent {
+    id: string
+    date: string
+    eventType: string
+}
+
+export const ContactsList = () => {
+    const [showCreateAContactForm, setShowCreateAContactForm] = useState<boolean>(false)
+    const [contacts, setContacts] = useState<Contact[]>([])
+
+    const toggleContactForm = () => {
+        setShowCreateAContactForm(!showCreateAContactForm)
+    }
+
+    const handleContactCreate = (newContact: Contact) => {
+        setContacts((prevState) => [...prevState, newContact])
+    }
+
+    useEffect(() => {
+        const getContacts = async () => {
+            try {
+                const response = await axios.get('http://localhost:8000/contacts')
```

Comment:
Was this routed created? If it is, is it returning contact data correctly?

Reply:
Yeah it is.

Reply:
I sent you the code on discord